src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -178,11 +178,41 @@ public function get_view_html() {
 		$context   = tribe_context();
 		$view_slug = $context->get( 'view' );
 
-		if (
+		/**
+		 * Filters the HTML for the view before we do any other logic around that.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $pre_html  Allow pre-filtering the HTML that we will boostrap.
+		 * @param string          $view_slug The slug of the View that will be built, based on the context.
+		 * @param \Tribe__Context $context   Tribe context used to setup the view.
+		 * @param \WP_Query       $query     The current WP Query object.
+		 */
+		$pre_html = apply_filters( 'tribe_events_views_v2_bootstrap_pre_get_view_html', null, $view_slug, $query, $context );
```

Comment:
Should we have another filter, view based?

Reply:
I wanted to avoid doing because that in on the bootstraping, I wanted to make it very generic, view specific filters only inside of the `View` class.